presto-orc/src/main/java/com/facebook/presto/orc/stream/DecimalOutputStream.java

```diff
@@ -110,9 +110,9 @@ public void close()
     }
 
     @Override
-    public StreamDataOutput getStreamDataOutput(int column)
+    public StreamDataOutput getStreamDataOutput(int column, int sequence)
     {
-        return new StreamDataOutput(buffer::writeDataTo, new Stream(column, DATA, toIntExact(buffer.getOutputDataSize()), true));
+        return new StreamDataOutput(buffer::writeDataTo, new Stream(column, DATA, toIntExact(buffer.getOutputDataSize()), true, sequence, Optional.empty()));
```

Comment:
why is the Stream constructor using the parameter with Optional.empty() for offset, it can use the newly added constructor which would does not require the offset parameter. Same for all other new places as well.